src/swephR.cpp

```diff
@@ -611,14 +740,27 @@ Rcpp::List azalt_rev(double jd_ut, int coord_flag, Rcpp::NumericVector geopos, R
   return Rcpp::List::create(Rcpp::Named("xout") = xout);
 }
 
+//' @details
+//' \describe{
+//' \item{swe_refrac()}{Calculate either the topocentric altitude from the apparent altitude
+//' or the apparent altitude from the topocentric altitude.
+//' }
```

Comment:
The brace opened on line 744 is never closed. This is the reason for the warning. It is probably best to add a closing brace in line 746, similar to other uses of this documentation pattern.